functions.php

```diff
@@ -1385,90 +1385,91 @@ function wp_autoupdates_disable_auto_updates() {
-					update_site_option( 'wp_auto_update_themes', $wp_autoupdate_themes );
-
-					/** This filter is documented in wp-admin/includes/class-wp-plugins-list-table.php */
-					$all_themes           = wp_get_themes();
-					$wp_autoupdate_themes = array_intersect( $wp_autoupdate_themes, array_keys( $all_themes ) );
-					$enabled_count        = count( $wp_autoupdate_themes );
-					$theme_name           = $all_themes[ $asset ]->get( 'Name' );
-
-					$theme_url = add_query_arg(
-						array(
-							'action'   => 'autoupdate',
-							'theme'    => $asset,
-							'_wpnonce' => wp_create_nonce( 'autoupdate-theme_' . $asset ),
-						),
-						'themes.php'
-					);
-
-					$return_html = sprintf(
-						'<p class="theme-autoupdate-disabled"><a href="%s" class="edit theme-autoupdate-enable" aria-label="%s"><span class="theme-autoupdate-label">%s</span></a></p>',
-						esc_url_raw( $theme_url ),
-						esc_html__( 'Enable automatic updates for', 'wp-autoupdate' ) . ' ' . esc_html( $theme_name ),
-						esc_html__( 'Enable auto-updates', 'wp-autoupdates' )
-					);
-
-					wp_send_json_success(
-						array(
-							'enabled_count'  => '(' . absint( $enabled_count ) . ')',
-							'disabled_count' => '(' . absint( count( $all_themes ) - $enabled_count ) . ')',
-							'return_html'    => wp_kses_post( $return_html ),
-							'type'           => 'theme',
-						)
-					);
-					break;
-				}
-			}
-		}
+		$wp_autoupdate_themes = (array) get_site_option( 'wp_auto_update_themes', array() );
+		$wp_autoupdate_themes = array_diff( $wp_autoupdate_themes, array( $asset ) );
+		update_site_option( 'wp_auto_update_themes', $wp_autoupdate_themes );
+
+		/** This filter is documented in wp-admin/includes/class-wp-plugins-list-table.php */
+		$all_themes           = wp_get_themes();
+		$wp_autoupdate_themes = array_intersect( $wp_autoupdate_themes, array_keys( $all_themes ) );
+		$enabled_count        = count( $wp_autoupdate_themes );
+
+		$theme_url = add_query_arg(
+			array(
+				'action'   => 'autoupdate',
+				'theme'    => $asset,
+				'_wpnonce' => wp_create_nonce( 'autoupdate-theme_' . $asset ),
+			),
+			'themes.php'
+		);
+
+		$aria_label = esc_attr(
+			sprintf(
+				/* translators: Theme name. */
+				_x( 'Enable automatic updates for %s', 'plugin', 'wp-autoupdates' ),
+				esc_html( $all_themes[ $asset ]->get( 'Name' ) )
+			)
+		);
+
+		$return_html = sprintf(
+			'<p class="theme-autoupdate-disabled"><a href="%s" class="edit theme-autoupdate-enable" aria-label="%s"><span class="theme-autoupdate-label">%s</span></a></p>',
+			esc_url_raw( $theme_url ),
+			$aria_label,
+			esc_html__( 'Enable auto-updates', 'wp-autoupdates' )
+		);
+
+		wp_send_json_success(
+			array(
+				'enabled_count'  => '(' . absint( $enabled_count ) . ')',
+				'disabled_count' => '(' . absint( count( $all_themes ) - $enabled_count ) . ')',
+				'return_html'    => wp_kses_post( $return_html ),
+				'type'           => 'theme',
+			)
+		);
 	}
+
 	wp_send_json_error(
```

Comment:
This code doesn't seem reachable unless `type` is not set, but that makes the request invalid?